src/cleed++/Util/teestream.cc

```
@@ -0,0 +1,7 @@
+#include "teestream.hh"
+
+teestream::teestream(std::ostream & o1, std::ostream & o2)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: tbuf

(cppcoreguidelines-pro-type-member-init)